.ci/benchmarks.groovy

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env groovy
+@Library('apm@current') _
+
+pipeline {
+  agent { label 'linux && immutable' }
+  environment {
+    REPO = 'apm-server'
+    BASE_DIR = "src/github.com/elastic/${env.REPO}"
+    BRANCH_NAME_LOWER_CASE = "${env.BRANCH_NAME.toLowerCase().replaceAll('[^a-z0-9-]', '-')}"
+    AWS_ACCOUNT_SECRET = 'secret/observability-team/ci/elastic-observability-aws-account-auth'    
+    EC_KEY_SECRET = 'secret/observability-team/ci/elastic-cloud/observability-pro'
+    BENCHMARK_ES_SECRET = 'secret/apm-team/ci/benchmark-cloud'
+    TERRAFORM_VERSION = '1.1.9'
+    
+    CREATED_DATE = "${new Date().getTime()}"
+    JOB_GCS_BUCKET_STASH = 'apm-ci-temp'
+    JOB_GCS_CREDENTIALS = 'apm-ci-gcs-plugin'
+  }
+
+  options {
+    timeout(time: 3, unit: 'HOURS')
+    buildDiscarder(logRotator(numToKeepStr: '100', artifactNumToKeepStr: '30', daysToKeepStr: '30'))
+    timestamps()
+    ansiColor('xterm')
+    disableResume()
+    durabilityHint('PERFORMANCE_OPTIMIZED')
+    rateLimitBuilds(throttle: [count: 60, durationName: 'hour', userBoost: true])    
+  }
+
+  stages {
+    stage('Checkout') {
+      options { skipDefaultCheckout() }
+      steps {
+        deleteDir()
+        gitCheckout(basedir: "${BASE_DIR}", shallow: false)
+      }
+    }
+
+    stage('Benchmarks') {
+      options { skipDefaultCheckout() }
+      environment {
+        SSH_KEY = "./id_rsa_terraform"
+        TF_VAR_private_key = "${env.SSH_KEY}"
+        TF_VAR_public_key = "${env.SSH_KEY}.pub"
```

Comment:
these environment variables are not masked on the CI logs

Reply:
That's not a problem, is it?

Reply:
there is a possibility of leaking it so I'd consider it an issue.

Reply:
ok, I thought was the actual key